backend/src/Designer/Controllers/ProcessModelingController.cs

```diff
@@ -109,6 +109,33 @@ public async Task<IActionResult> UpsertProcessDefinitionAndNotify(string org, st
             return Accepted();
         }
 
+        [HttpPut("data-type")]
+        public async Task<IActionResult> ProcessDataTypeChangedNotify(string org, string repo, [FromForm] string metadata, CancellationToken cancellationToken)
```

Comment:
Metadata should be used as strong type in method argument of the string + deserialization